src/cli/cli_dataset.cpp

```diff
@@ -609,6 +610,10 @@ otError Dataset::Print(otOperationalDatasetTlvs &aDatasetTlvs)
 
     for (const ComponentTitle &title : kTitles)
     {
+        if (aNonsensitiveOnly && title.mIsSensitive)
+        {
+            continue;
+        }
```

Comment:
Per code style the declarations should be in front of this.

Reply:
Done.